core/src/packet_deserializer.rs

```diff
@@ -82,19 +96,26 @@ impl PacketDeserializer {
         &self,
         recv_timeout: Duration,
         packet_count_upperbound: usize,
-    ) -> Result<(Vec<PacketBatch>, Option<SigverifyTracerPacketStats>), RecvTimeoutError> {
+    ) -> Result<
+        (
+            usize,
+            Vec<BankingPacketBatch>,
+            Option<SigverifyTracerPacketStats>,
```

Comment:
Confused what's happening here - why are there tracer packet stats inside and outside of BankingPacketBatch?

Reply:
good point. this ended up in another nice cleanup: 0952b0db03cd3b046627f268b8c5b74eba5f609d